banzai_nres/traces.py

```diff
@@ -60,6 +59,9 @@ def refine_traces(traces, image, weights=None):
 
 
 class TraceInitializer(Stage):
+    # Minimum half width of a feature to be considered a trace
+    min_trace_half_width = 500
+
```

Comment:
moved the module level variable to inside the class.

Reply:
we use fitting in serval places. seems like ti would be good to keep some of this separate.

Reply:
Why?

Reply:
Also constants are generally all caps.

Reply:
I do not quite understand the request, sorry. Where should I move this?

Reply:
Oh now I see the comment. I moved this because the unit tests require this to be less than the image size, so it was set to 50. But for NRES this should be large since every trace > 1000 pixels wide. It concerned me that a feature with a size of 51 pixels wide would pass the check in production. Moving it to inside the class allows me to set this to 50 in the unit test, without overriding a module level variable upon starting the test.

Reply:
ah ok.